citation_helper_service/tests/test_internals.py

```diff
@@ -72,7 +72,7 @@ def test_config_values(self):
                     "CITATION_HELPER_SOLR_PATH",
                     "DISCOVERER_PUBLISH_ENDPOINT",
                     "DISCOVERER_SELF_PUBLISH"]
-        missing = [x for x in required if x not in self.app.config.keys()]
+        missing = [x for x in required if x not in list(self.app.config.keys())]
```

Comment:
I think this `list(...)` (automatically added by the porting tools) might not be needed here.